cmd/create/dependabot/flag.go

```diff
@@ -30,7 +31,7 @@ func (f *flag) Validate() error {
 		}
 	}
 
-	{
+	if !file.Exists(".github/CODEOWNERS") {
 		if len(f.Reviewers) == 0 {
 			return tracer.Mask(runtime.InvalidFlagError, tracer.Context{Key: "reason", Value: "-r/--reviewers must not be empty"})
 		}
```

Comment:
There might be cases where a CODEOWNERS file already exists, which means for us that we do not need to accept the reviewers flag anymore, because those are already managed and `workflow` does not need to bootstrap this file.